src/sidebar/components/Annotation/Annotation.tsx

```diff
@@ -35,7 +35,7 @@ function SavingMessage() {
       <span
         // Slowly fade in the Spinner such that it only shows up if the saving
         // is slow
-        className="text-xl animate-fade-in-slow"
+        className="text-[16px] animate-fade-in-slow"
```

Comment:
The `Spinner` component for better or worse is em-sized. At `size="small"` it will size itself to `1em` square (documented in the pattern library). It was designed this way because spinners are commonly used along with text, so sizing a spinner relative to local text makes sense, at least some of the time.

Here there's no local text. Our desired outcome is a spinner that is 16px square. So I am still setting a text size here, but decoupling it from our type scale and trying to be clear about the intent. Put another way, if the text scale values change, I still want this spinner to be 16px square.